core/api/serializers.py

```
@@ -2,11 +1,208 @@
+    """
+    videos = ContributionVideoSerializer(many=True, required=False)
+    tags = ContributionTagSerializer(many=True, required=False)
+    origine = ContributionOriginSerializer(many=True, required=False)
+    notes = ContributionNoteSerializer(many=True, required=False)
+    is_enrolled = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Contributions
+        fields = '__all__'
+
+    def get_is_enrolled(self, obj):
```

Ruff (0.8.2)

Missing return type annotation for public function `get_is_enrolled`

(ANN201)